src/main/java/com/thoughtworks/pacman/ui/screens/WinScreen.java

```diff
@@ -16,15 +20,46 @@
     private final Game game;
     private boolean startGame;
 
+    private Color buttonMainColor; // main color theme: Yellow
```

Comment:
Constructorda atamak yerine doğrudan burada ata

buttonOnHoverColor için de aynı.

```new Color(255, 255, 0)``` yerine  ```Color.YELLOW``` kullan.